apiclient/exceptions.py

```diff
@@ -59,3 +66,24 @@ class UnexpectedError(APIRequestError):
     """An unexpected error occurred when making the request."""
 
     pass
+
+
+class BaseExceptionHandler:
```

Comment:
```suggestion
class BaseExceptionHandler:
    """Translates a response into an apiclient exception."""
```
Needs some docstrings.

Reply:
👌